test/extended/util/framework.go

```diff
@@ -40,13 +40,79 @@ var TestContext e2e.TestContextType
 
 const pvPrefix = "pv-"
 
+//FixOpenShiftNamespaceImageStreams is a temporary workaround for the intermittent
+// issue seen in extended tests where *something* is deleteing the pre-loaded, languange
+// imagestreams from the OpenShift namespace
+func FixOpenShiftNamespaceImageStreams(oc *CLI) {
```

Comment:
I was very hesitant to add logging in the mainline rest path, and wanted to do save that as a last resort.  And to reiterate, this workaround is going to be short term (I want to confirm that the better results I'm seeing in my pull with it translate to the jenkins job).

That said, enabling audit logging sounds promising to me though.  However, I'm having trouble finding how to do that in the documentation - any pointers?

thanks